spectral/scripts/rules.sh

```diff
@@ -7,7 +7,23 @@
 
 echo "fetching kas-fleet-manager from main" 
 wget -P .openapi https://raw.githubusercontent.com/redhat-developer/app-services-sdk-js/main/.openapi/kas-fleet-manager.yaml
+echo "fetching srs from main" 
+wget -P .openapi https://raw.githubusercontent/redhat-developer/app-services-sdk-js/main/.openapi/srs-fleet-manager.json
 
-## TODO add more apis
+# updates made to the following need to be refected in their respective .openapi files before the linting can pass
```

Comment:
??

Reply:
I think it would be good to have single json file on the sdk that we can fetch to get all sdks: 

https://github.com/redhat-developer/app-services-sdk-go/blob/main/.config/api-client-metadata.json

then we can make this check generic - fetching all data

Reply:
so these are the other OAS files that we look after, I've tested them as is and they currently fail, but when my changes to their OAS files land, the following can be uncommented, so its set up to add them all. 
I am hopeful of those changes landing in the next day or two so that's the reason I just left them commented out

Reply:
I've addressed the comment 😄 

is that `api-client-metadata.json` file auto generated? is that the file you'd
sure, as soon as we get all of those apis passing, I can edit this to pull from there?

Reply:
I think that is idea for future. What we have right now is good.